optimizely/optimizely.py

```diff
@@ -629,8 +632,11 @@ def get_variation(
             return None
 
         user_context = OptimizelyUserContext(self, self.logger, user_id, attributes, False)
-
-        variation, _ = self.decision_service.get_variation(project_config, experiment, user_context)
+        user_profile_tracker = user_profile.UserProfileTracker(user_id, self.user_profile_service, self.logger)
```

Comment:
should there be a `user_profile_service.load_user_profile()` call here. I suspect this could be a reason behind FSC failures